stage1/init/common/mount.go

```diff
@@ -45,13 +56,28 @@ func IsMountReadOnly(vol types.Volume, mountPoints []types.MountPoint) bool {
 	return isMPReadOnly(mountPoints, vol.Name)
 }
 
-// GenerateMounts maps MountPoint paths to volumes, returning a list of Mounts.
-func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume) []schema.Mount {
+func convertedFromDocker(ra *schema.RuntimeApp) bool {
+	ann := ra.Annotations
```

Comment:
@iaguis Bad news, not working for rktnetes still, as it's using `run --pod-manifest`... When running from the `run --pod-manifest`, we don't really have the annotations in the runtime app. So probably we should check the image's annotation instead?